R/SSplotHCxval.R

```diff
@@ -1,81 +1,63 @@
 #' Hindcasting Cross-Validations of Multiple Models
 #'
-#' Plots one-step ahead hindcasting cross-validations and computes MASE from prediction residuals.
-#' MASE is calculated as the average ratio of mean absolute error (MAE) of prediction residuals (MAE.PR) and naive predictions (MAE.base)
-#' MASE.adj sets the MAE.base to a minimum MAE.base.adj (default=0.1)
-#' MASE.adj allow passing (MASE<1) if MAE.PE < 0.1 and thus accurate if obs show very little annual variation
+#' Plots one-step ahead hindcasting cross-validations and computes MASE from
+#' prediction residuals.`MASE` is calculated as the average ratio of mean
+#' absolute error (`MAE`) of prediction residuals (`MAE.PR`) and naive
+#' predictions (`MAE.base`). `MASE.adj` sets the `MAE.base` to a minimum
+#' `MAE.base.adj` (`default=0.1`). `MASE.adj` allow passing (`MASE<1`) if
+#' `MAE.PE < 0.1` and thus accurate if obs show very little annual variation
 #'
-#' @param retroSummary List created by r4ss::SSsummarize() or ss3diags::SSretroComps()
-#' @param subplots optional use of c("cpue","len","age"), yet to be tested for age.
-#' @param models Optional subset of the models described in
-#' r4ss function summaryoutput().  Either "all" or a vector of numbers indicating
-#' columns in summary tables.
-#' @param Season option to specify Season as an integer of value 1-4 - Default uses first available, i.e. usual Seas = 1
+#' @param retroSummary List created by [r4ss::SSsummarize()] or
+#' [ss3diags::SSretroComps()]
+#' @param subplots optional use of the following:
+#' \itemize{
+#'  \item `"cpue"` Index data
+#'  \item `"len"` Length composition data
+#'  \item `"age"` Age composition data (yet to be tested)
```

Comment:
Age composition data has been tested and works, can remove the 'yet to be tested'